### ISSUE TITLE
Start a HackerOne bug bounty program.

### ISSUE THREAD

**Issue by EdOverflow** [liberapay/liberapay.com#549]
HackerOne Professional is now free for open source projects: https://www.hackerone.com/blog/HackerOne-Professional-Free-For-Open-Source-Projects

**Comment by EdOverflow** [comment]
@Changaco: Would you like to go ahead and set up a VRP on [Hackerone](https://hackerone.com/teams/new)? I will help write a nice policy and triage reports.

**Comment by Changaco** [comment]
I'm also about being distracted by both noise and genuine reports. I would also prefer having fewer [known issues](https://github.com/liberapay/liberapay.com/labels/Self-defense) before asking the community to help us find more.

**Comment by EdOverflow** [comment]
@Changaco: From experience, no matter what you do, there is very little chance that you will prevent noise. I agree that some of the `Self-defense` tickets should be resolved before we start, but I would not worry too much about noise.

**Comment by EdOverflow** [comment]
I forgot to clarify my terminology. We need to discuss whether you want to launch a **bug bounty program** (BBP) or a **vulnerability disclosure program** (VDP). The former would require financially rewarding researchers that report valid security issues to Liberapay. The latter is basically a security@ address on a platform such as HackerOne.

**Comment by Changaco** [comment]
Using the organization's funds requires consensus among the codirectors. Want to [open an issue in the org repo](https://github.com/liberapay/liberapay.org/issues/new)?

**Comment by EdOverflow** [comment]
I am still up for managing this. Since I work for HackerOne now, I can probably get things sorted out fairly quickly. It would be nice to have an inbox that all project maintainers can keep an eye on — a platform seems ideal compared to an email inbox.

**Comment by Changaco** [comment]
@EdOverflow I've created the program and invited you. I guess the next step is to figure out bounty amounts, then we should be ready to launch.

**Comment by Changaco** [comment]
Done: <https://hackerone.com/liberapay>.

**Comment by EdOverflow** [comment]
Fantastic work! This is just what we needed.

**Comment by Changaco** [comment]
For the record, a small illustration of the negative side of opening a program on HackerOne:

![Screenshot of traffic stats from Cloudlfare](https://user-images.githubusercontent.com/1581590/41088430-d7ac8f8a-6a3f-11e8-9026-8a47cde930c6.png)

Without Cloudflare's "Under Attack" mode all those requests would have hit the origin server.

Related comments: <https://github.com/liberapay/salon/issues/218#issuecomment-394144676>.
